src/rp2040/hardware_regs/include/hardware/platform_defs.h

```diff
@@ -46,7 +46,7 @@
 
 #define HAS_SIO_DIVIDER 1
 #define HAS_RP2040_RTC  1
-// PICO_CONFIG: XOSC_HZ, Crystal oscillator frequency in Hz, type=int, default=12000000, advanced=true, group=hardware_base
+// PICO_CONFIG: XOSC_HZ, The crystal oscillator frequency in Hz, type=int, default=12000000, advanced=true, group=hardware_base
```

Comment:
Seems like we wre removing "The" in general - i assume you did this to match RP2350 perhaps, but we should probably change that instead

Reply:
Haha. When there were descriptions that only differed slightly, I wasn't sure which direction to go in, so I assumed that as RP2350 was the "most recently edited" version, then that was therefore "the most correct". But sure, I'll tack another commit on the end here to get rid of the "The..." versions :slightly_smiling_face: